dataclasses is integrated in Python 3.7 and hence is no longer required
Issue fixed in master branch of PyTorch

Author: Alexander Grund (TU Dresden)

diff --git a/setup.py b/setup.py
index e599d82898..500186a7a4 100644
--- a/setup.py
+++ b/setup.py
@@ -337,7 +337,7 @@ def build_deps():
 ################################################################################
 
 # the list of runtime dependencies required by this built package
-install_requires = ['future', 'typing_extensions', 'dataclasses']
+install_requires = ['future', 'typing_extensions']
 
 missing_pydep = '''
 Missing build dependency: Unable to `import {importname}`.
